pyfolio/utils.py

```diff
@@ -381,190 +374,69 @@ def estimate_intraday(returns, positions, transactions, EOD_hour=23):
-    period : {'1MO', '3MO', '6MO', 1', '5', '10'}, optional
-        Which maturity to use.
-    Returns
-    -------
-    pd.Series
-        Annual treasury yield for every day.
-    """
-    return empyrical.utils.get_treasury_yield(start=None,
-                                              end=None,
-                                              period='3MO')
+    return clipped_rets
 
 
-@deprecated(msg=DEPRECATION_WARNING)
-def get_utc_timestamp(dt):
+def to_utc(df):
     """
-    Returns the Timestamp/DatetimeIndex
-    with either localized or converted to UTC.
-    Parameters
-    ----------
-    dt : Timestamp/DatetimeIndex
-        the date(s) to be converted
-    Returns
-    -------
-    same type as input
-        date(s) converted to UTC
+    For use in tests; applied UTC timestamp to DataFrame.
     """
-    return empyrical.utils.get_utc_timestamp(dt)
 
+    try:
+        df.index = df.index.tz_localize('UTC')
+    except TypeError:
+        df.index = df.index.tz_convert('UTC')
 
-@deprecated(msg=DEPRECATION_WARNING)
-def cache_dir(environ=environ):
-    return empyrical.utils.cache_dir(environ=environ)
+    return df
 
 
-@deprecated(msg=DEPRECATION_WARNING)
-def ensure_directory(path):
+def to_series(df):
     """
-    Ensure that a directory named "path" exists.
+    For use in tests; converts DataFrame's first column to Series.
     """
-    return empyrical.data_path(path)
-
-
-@deprecated(msg=DEPRECATION_WARNING)
-def data_path(name):
-    return empyrical.data_path(name)
-
 
-@deprecated(msg=DEPRECATION_WARNING)
-def _1_bday_ago():
-    return empyrical._1_bday_ago()
+    return df[df.columns[0]]
 
 
-@deprecated(msg=DEPRECATION_WARNING)
-def load_portfolio_risk_factors(filepath_prefix=None, start=None, end=None):
-    """
-    Load risk factors Mkt-Rf, SMB, HML, Rf, and UMD.
-    Data is stored in HDF5 file. If the data is more than 2
-    days old, redownload from Dartmouth.
-    Returns
-    -------
-    five_factors : pd.DataFrame
-        Risk factors timeseries.
-    """
-    return empyrical.utils.load_portfolio_risk_factors(filepath_prefix=None,
-                                                       start=None,
-                                                       end=None)
+# The following functions are all simply passthroughs to empyrical. With the
+# exception of get_utc_timestamp, they are all deprecated and will be removed
+# in a later version.
+default_returns_func = empyrical.utils.default_returns_func
+get_fama_french = empyrical.utils.get_fama_french
+get_returns_cached = empyrical.utils.get_returns_cached
+get_symbol_returns_from_yahoo = empyrical.utils.get_symbol_returns_from_yahoo
+get_treasury_yield = empyrical.utils.get_treasury_yield
+get_utc_timestamp = empyrical.utils.get_utc_timestamp
+cache_dir = empyrical.utils.cache_dir
+ensure_directory = empyrical.utils.ensure_directory
+data_path = empyrical.utils.data_path
+_1_bday_ago = empyrical.utils._1_bday_ago
```

Comment:
This looks unused in pyfolio. Do we think external consumers were depending on this?

Reply:
I couldn't say; I included it just in case though.

Reply:
Let's remove these.

Reply:
done